### ISSUE TITLE
Slider: range mode aria-valuenow isn't related to actual values

### ISSUE THREAD

**Issue by ribrown-bishs** [primefaces/primereact#4242]
### Describe the bug

When using the slider with step 5, min 10, max 50, I would expect that the aria-valuenow would reflect the steps' values when they are selected. However, the aria-valuenow is instead a copy of the `style="left: ... ` percentage. 
![image](https://user-images.githubusercontent.com/109183551/231197538-474ead87-49f3-4122-af08-19092c1321de.png)

![image](https://user-images.githubusercontent.com/109183551/231197597-33c7dbce-5adf-4b74-8ad4-bdf74a46d8e3.png)

### Reproducer

https://codesandbox.io/s/primereact-test-forked-31vw6s?file=/src/index.js

### PrimeReact version

9.2.2

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Steps to reproduce the behavior

1. Drag one of the handles of the slider
2. Inspect the slider handle element
3. View the aria-valuenow (as compared to the steps that should be there, between 10-50 by fives)

### Expected behavior

The handle's aria-valuenow value should be the same as the "selected" step value. In this example, if I drag the left side handle up 1 step, the value should be 15.

**Comment by melloware** [comment]
I actually think the bug is its not respecting the step either?  When I set `step={5}` on your example the first slide is to 12.5 not 5.

**Comment by ribrown-bishs** [comment]
Hm. That part seems to be working for me, I updated the sandbox to show the selected values to help.

**Comment by ribrown-bishs** [comment]
I don't think so because the `min` prop is set to 10.

**Comment by melloware** [comment]
but this is how its sets the initial vlaue.

```js
const value = props.range ? props.value || [props.min, props.max] : props.value || 0;
```

in your case props.value = 10,50 which would mean value is 10 for the low range?

**Comment by ribrown-bishs** [comment]
Sorry, you're correct, I misunderstood. 
Yeah, if you're seeing 0 in there - when the min is set AND the initial lower value is set - then something seems wrong. 
I haven't seen that, but I have seen that the `aria-valuenow` isn't set until the user interacts with the slider.

**Comment by melloware** [comment]
Yep definitely a bug or bug(s) here with Range mode.

**Comment by ribrown-bishs** [comment]
For the aria issue, seems like around this line the left/bottom value could be replaced by the props.value[0/1]?
https://github.com/primefaces/primereact/blob/26d68171a3af9649d4a1880af126d7214bd79ea9/components/lib/slider/Slider.js#L203

**Comment by melloware** [comment]
Here is how those values are calculated.

```js
const handleValueStart = ((value[0] < props.min ? props.min : value[0] - props.min) * 100) / (props.max - props.min);
const handleValueEnd = ((value[1] > props.max ? props.max : value[1] - props.min) * 100) / (props.max - props.min);
```
